### DISCUSSION TITLE
Feb 13

### DISCUSSION THREAD

**Discussion by miniposi** [GymHubCommunity/GymHub-FE#148] [DailyMeeting]
### ⚽️ 회의 내용

>🎤 오늘의 mc  윤혁

### 개발 현황
슬희: 회원가입 페이지 하는중, refresh token 빼고 나머지 다 함!
수: 운동 기록 페이지 완료, 새로운 거 찾으러
수영: 인터셉터 끝 / 좋아요 컴포넌트 시작
윤혁: 메인 페이지 디자인 90프로

### 오늘의 바보 같은 점 고백
궁수: 전 이제야 깨달아버렸어요 사실 전 '운동 기록 _와이어프레임_'만 하면 된다는 걸 ....

### 🏀 오늘 할 일
슬희: refresh token 끝내기
수: 팔로우 도전
수영: pre-signed url 도전
윤혁: 메인 페이지 진짜 끝내기

### 🎱 TMI
슬희: 스타필드 엄청 좋은 헬스장 생김 🏋🏻 현재 다니는 헬스장 환불 받고, 스타필드 헬스장으로 가볼까 생각중
수: 너무 추운거 같다 🥶
수영: 내일 모레 친한 친구 국밥집 오픈 😎 뼈 해장국 사장님 친구;; 수영님 아지트가 될 예정 친구들끼리 돈 모아서 공기 청정기 선물
윤혁: 지갑이 왔는데 마음에 든다 + 바지 또 구매

**Comment by hanseulhee** [comment]
난 미래에 마라샹궈 사장님 하겠서요
